src/Experiment/src/AssemblyBuilder.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
```

Comment:
```suggestion
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.
﻿
﻿﻿﻿using System;
```
NIT: All *.cs files should have the License header on top